corehq/apps/widget/forms.py

```diff
@@ -0,0 +1,95 @@
+from django.utils.translation import ugettext_lazy as _
+from corehq.apps.widget.models import DialerSettings
+
+from corehq.apps.hqwebapp import crispy as hqcrispy
+from crispy_forms import layout as crispy
+from crispy_forms.layout import Submit
+
+from memoized import memoized
+from django import forms
+
+
+class DialerSettingsForm(forms.ModelForm):
+    is_enabled = forms.BooleanField(
+        label=_("Enable AWS Connect Dialer"),
+        required=False
+    )
+    aws_instance_id = forms.CharField(
+        label=_('AWS Instance ID'),
+        help_text=_("""Enter "yourinstance" if your AWS Connect account is
+                        "https://yourinstance.awsapps.com/connect/" """)
+    )
+
+    dialer_page_header = forms.CharField(
+        label=_('Dialer Page Title'),
+        help_text=_("A title for the Dialer Page header")
+    )
+    dialer_page_subheader = forms.CharField(
+        label=_('Dialer Page Subtitle'),
+        help_text=_("A subtitle for the Dialer Page header")
+    )
+
+    class Meta:
+        model = DialerSettings
+        fields = [
+            'aws_instance_id',
+            'dialer_page_header',
+            'dialer_page_subheader'
+        ]
+
+    def __init__(self, data, *args, **kwargs):
+        self._domain = kwargs.pop('domain')
```

Comment:
Nitpick: why `_domain` instead of `domain`? Most forms use the latter.

Reply:
Just boilerplate from somewhere else. Will change.